### PR TITLE
Add release notes for 5.0.0

### PR DESCRIPTION
**Pull request by edthedev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,4 +41,5 @@ Fake new release for CI/CD Demo.
 
 ### Added
 
-- Initial release
+Initial Release
```
Comment (edthedev) [inline CHANGELOG.md:44]:
```suggestion
- Initial release
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,27 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Add duration, source and why parameters (#16)
+
+### Updated
+
+- Update dependencies for security fix (#26)
+
+## [0.0.0] - 2023-02-17 [YANKED]
+
+Untagged production release. Latest as of 2023-9-10.
+
```
Comment (edthedev) [inline CHANGELOG.md:25]:
```suggestion
Do not use unless rolling back from 5.0.0.
````

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,7 +7,27 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Add play_count test (#47)
+- Add autoscale parameter to null router 'block' action (#22)
+- Add duration, source and why parameters (#16)
+
+### Updated
+
+- Update dependencies for security fix (#26)
+
+## [0.0.0] - 2023-02-17 [YANKED]
```
Comment (edthedev) [inline CHANGELOG.md:22]:
```suggestion
## [0.0.0] - 2023-02-17
```

Not 'YANKED' since this is a valid rollback target.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,7 +7,28 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+- Add autoscale parameter to null router 'block' action (#22)
+- Add duration, source and why parameters (#16)
+
+### Updated
+
+- Update dependencies for security fix (#26)
+
+## [0.0.0] - 2023-02-17
+
+Untagged production release. Latest as of 2023-9-10.
```
Comment (edthedev) [inline CHANGELOG.md:24]:
```suggestion
```

**Review by ddriddle — APPROVED** [review]

**Review by zdc217 — APPROVED** [review]

**Review by tzturner — APPROVED** [review]

**Review by mpitcel — APPROVED** [review]

**Pull request merged**